components/docs-chef-io/content/automate/datafeed.md

```diff
@@ -33,37 +29,241 @@ A Data Feed operates by doing the following:
+1. Select **New Integration**
 
-![Setup Data Feed Page](/images/automate/filled_form_create_data_feed.png)
+Currently, the data feed has two types of integrations:
 
-1. In the **Settings** tab, navigate to _Data Feeds_ in the sidebar
-1. Select **Create Data Feed**
-1. Enter a unique Data Feed name
-1. Enter the URL for your Data Feed endpoint, including any specific port details
-1. Enter the Username and Password that your 3rd party endpoint requires for authentication
-1. Select **Test Data Feed** to begin validating the connection details
-1. Once the test is successful, select **Create Data Feed** to save your Data Feed configuration
+- Webhook Integration
+  - ServiceNow
+  - Splunk
+  - ELK
+  - Custom
+- Storage Integration
+  - Minio
+  - Amazon S3
+
+{{< figure src="/images/automate/choose-a-data-feed-integration.png" alt="Choose a Data Feed Integration">}}
+
+## Webhook Integration
+
+Create a data feed instance using any webhook integration.
+
+### ServiceNow
+
+To add a Data Feed instance in Chef Automate using ServiceNow integration:
+
+1. Select **New Integration**.
+
+1. Select the **ServiceNow** icon under Webhook Integration.
+
+1. In the form, enter a unique **Data Feed name**.
+
+1. Enter a **URL** for the Data Feed endpoint, including any specific port details.
+
+1. Select an **authentication** from the couple of options in the drop-down.
+  
+  - If you select the **Username and Password** authentication, enter the **Username** and **Password** that your 3rd party endpoint requires for authentication. OR
+  - If the authentication is set to **Access Token**, set the **Token Type** and **Token** for the 3rd party authentication.
+
+1. Select **Test Connection** to start validating the connection details.
+
+1. Once the test is successful, select **Save** to save the Data Feed configuration.
+
+{{< figure src="/images/automate/data-feed-instance-using-servicenow-integration.png" alt="Data Feed Instance using ServiceNow Integration">}}
+
+#### Edit a ServiceNow Data Feed Instance
+
+To edit a Data Feed instance of ServiceNow Integration:
+
+1. Select the data feed instance name to open its detail page.
+
+1. Edit the Data Feed **Name** or **URL**.
+
+1. Select **Test Data Feed** button to test the URL.
+
+1. Select **Save** to save your changes.
+
+{{< figure src="/images/automate/details-of-data-feed-instance-using-servicenow-integration.png" alt="Details of Data Feed Instance using ServiceNow Integration">}}
+
+You can also **Enable**, **Disable** and **Delete** the instance from the buttons provided on the details page.
+
+#### Delete a ServiceNow Data Feed Instance
+
+To delete an individual instance:
+
+1. Selecting the ellipses icon {{< fontawesome class="fas fa-ellipsis-h" >}}.
+
+1. Select **Delete**.
+
+{{< figure src="/images/automate/delete-the-data-feed-instance-using-servicenow-integration.png" alt="Delete the Data Feed Instance using ServiceNow Integration">}}
+
+1. In the popup, select **Delete Data Feed**.
```

Comment:
```suggestion
1. Select **Delete Data Feed** to confirm the action.
```

Reply:
The point of this change is to focus on describing the actions that the user needs to take instead of describing the UI.